spec/spec.md

```diff
@@ -597,6 +597,8 @@ While ION supports any public key JWK representation in a DID Document, implemen
 |--------|-------------|
 |secp256k1|ES256K|
 |Ed25519|EdDSA|
+|P-256 (secp256r1)|ES256|
+|P-384 (secp384r1)|ES384|
```

Comment:
Suggest we make P curves optional for both verifiers and wallets.
and suggest we enhance the table as below:

|Key Type|JWT Algorithm|Mandatory/Optional for the Verifier|Mandatory/Optional for the Wallet|
|--------|-------------|---|---|
|secp256k1|ES256K|M|O*|
|Ed25519|EdDSA|M|O*|
|P-256 (secp256r1)|ES256|O|O|
|P-384 (secp384r1)|ES384|O|O|
* MUST support at least one.

Reply:
also this sentence should be improved "While ION supports any public key JWK representation in a DID Document, implementors of this document MUST support JWT signature verification with the following Key Types and must support JWT signing with at least one." 
to something like
"Verifiers and wallets compliant with this profile MUST support JWT signature verification of the Key Types as defined in the table below"

Reply:
this change has not been addressed